firedrake/mg/embedded.py

```diff
@@ -10,7 +10,7 @@
 __all__ = ("TransferManager", )
 
 
-native_families = frozenset(["Lagrange", "Discontinuous Lagrange", "Real", "Q", "DQ", "BrokenElement", "Crouzeix-Raviart"])
+native_families = frozenset(["Lagrange", "Discontinuous Lagrange", "Real", "Q", "DQ", "BrokenElement", "Crouzeix-Raviart", "Kong-Mulder-Veldhuizen"])
```

Comment:
Is there any way that we can detect this family instead of enumerating them?

Reply:
We are inspecting UFL elements, so we need to compare strings.

In theory we could detect elements that only have point evaluation dofs if we were inspecting the finat element instead.